Solutions/Forescout Host Property Monitor/Playbooks/ReadMe.md

```diff
@@ -0,0 +1,68 @@
+# ForeScout playbook templates
+
+<img src="./forescout-logo.png" alt="drawing" width="20%"/><br>
+
+## Table of Contents
+
+1. [Overview](#overview)
+1. [Playbook Templates](#playbooks)
+1. [Known issues and limitations](#limitations)
+
+<a name="overview">
+
+# Overview
+
+"Forescout Host Property Monitor" is a project that helps to check and report IP addresses involved in various categories of malicious attacks by analyzing host properties uploaded by Forescout Continuum platform. This project analyzes number of times certain events are generated by the host to determine the classificaction and severity of activity by the host. It forwards that severity information back to Forescout Continuum platform through description field fo Sentinel Incident so that Coninuum platform can act upon the host.
+
+<a name="playbooks">
+
+# Playbook Templates
+This package includes:  
+
+* DNS Sniff Event Playbook   
+This topic describes a sample Logic App/Playbook, "DNS Sniff Event Analysis".
+Use the Analytic Rule to check what kind of incident is created and how many times the events happened for any endpoint within the time interval. Based on this information, the Logic App/Playbook will set the Description field of the incident with the following information:  
+  -	action_group: Indicates the category of action. The valid values are manage, notify, remediate, and restrict.  
+  -	disposition: Represents the actual action in 2-digit format. For example, 00 for Recheck Host, 10 for HTTP Notification.  
+  -	endpoint_ip: Indicates the IP address of the endpoint, which can be an IPv4 or IPv6 address.  
+  -	em_ip: Indicates the IP address of the Enterprise Manager to which the endpoint belongs.  
+  -	info: Indicates addistional information   
+The em_ip and info are optional.  
+
+Note that the eyeExtend for Microsoft Sentinel module polls the incidents information periodically and examines the Description field of each incident. Based on the information set in the Description field, the Continuum Platform applies the requested action set by Microsoft Sentinel to the endpoints.  
+The following is a sample description field of an incident set by the playbook:  
+action_group="restrict";disposition="31";endpoint_ip=["10.16.141.130"];em_ip=["10.16.147.96"];info=Generated_by_Dnsniff_event  
+Each action has a unique disposition value. The action name, disposition value, and action group supported by eyeExtend for Microsoft Sentinel are as follows:  
+
+Recheck Host	00	manage  
```

Comment:
Can we represent this information in form of a table. This would make the data more readable.